sdk/servicebus/Azure.Messaging.ServiceBus/tests/Samples/Sample11_CloudEvents.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace Azure.Messaging.ServiceBus.Tests.Samples
+{
+    public class Sample11_CloudEvents : ServiceBusLiveTestBase
+    {
+        [Test]
+        public async Task RoundTripCloudEvent()
+        {
+            await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitioning: false, enableSession: false))
+            {
+                #region Snippet:ServiceBusCloudEvents
+#if SNIPPET
+                string connectionString = "<connection_string>";
+                string queueName = "<queue_name>";
+#else
+                string connectionString = TestEnvironment.ServiceBusConnectionString;
+                string queueName = scope.QueueName;
+#endif
+                // since ServiceBusClient implements IAsyncDisposable we create it with "await using"
```

Comment:
```suggestion

                // since ServiceBusClient implements IAsyncDisposable we create it with "await using"
```

This is the only comment in the sample not preceded by a blank line and read a bit cramped to me on the markdown.